.github/workflows/remote-build-ios-hybrid.yml

```diff
@@ -0,0 +1,32 @@
+name: Remote Build iOS Hybrid
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: macos-15-xlarge
+    steps:
+      - name: Checkout
+        # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/composite/setupNode
+
+      - name: RNEF Remote Build - iOS
+        uses: callstackincubator/ios@v1
```

Comment:
should we also restrict the version here to a commit hash? Happy to do that

Reply:
Actually, I'll proceed with that as we don't leave it as is for any other action - especially critical one.